### PR TITLE
[Fix] Remove usernames and Wiki Challenge Solution Template reference in guide articles

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

You will notice I did not check off the 5th checkbox above.  The reason is, this PR affects multiple language's Guide articles, but technically, I am removing content and not adding content, so there is no need for translation review (as is the primary reason for this checkbox).

This PR came about based on noticing a Github username was added in another PR (#34858).  Please read the comments on this other PR for background on why my PR was created. 

Basically, it removes 3 types of content on applicable curriculum challenge articles:

1. references to `Add your username` at the bottom of the article
2. references to the `Wiki Challenge Solution Template`
3. references to `Thanks @username`

I have added the `blocked` label to this PR, because we want to make sure we review the other PRs which may affect these same files first.  Once all of them have been reviewed (closed or approved/merged), then I will rebase against master and resolve any merge conflicts.

There may be some other references to the `Wiki Challenge Solution Template` which have been translated, so I will continue to push new commits to this PR as I find additional references.

@raisedadead This is the PR I mentioned to you earlier today.

### CODE REVIEW

**Comment by RandellDawson** [comment]
I am closing this PR and breaking it up into separate PRs for each language.
